wordpress-mobile/WordPressAuthenticator-iOS
# Remove segue .showLoginEmail

Fixes #205
Ref. #182  

This PR removes all references to the segue `.showEmailLogin` and programmatically navigates the user to the `LoginEmailViewController`. There are 3 areas where the segues have been removed: 
1. WPiOS sign up
2. WPiOS sign in
3. WCiOS sign in

### To Test - WPiOS
1. Check out this branch
2. `bundle exec pod install` and ensure there are no errors
3. Build and run (there should be no errors)
4. Visit the WPiOS PR to run the changes: https://github.com/wordpress-mobile/WordPress-iOS/pull/13669

### To Test - WCiOS
1. Check out this branch
2. `bundle exec pod install`
3. Build and run (there should be no errors)
4. Visit the WCiOS PR to run the changes: https://github.com/woocommerce/woocommerce-ios/pull/2007
